src/main/java/frc/robot/subsystems/BallChucker9000.java

```diff
@@ -20,57 +20,68 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.Encoder;
-import edu.wpi.first.wpilibj.Solenoid;
-import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.controller.PIDController;
 
 public class BallChucker9000 extends SubsystemBase {
 
+    // PID
+    private final PIDController rotatorPID;
+    private final PIDController flywheelPID;
+
     // ESC declarations
-    private final VictorSP rotatorMotor;
-    private final VictorSP flywheelMotor;
-    private final VictorSP indexerMotor;
+    private final SpeedController rotatorMotor;
+    private final SpeedController rightFlywheelMotor;
+    private final SpeedController leftFlywheelMotor;
+    private final SpeedController indexerMotor;
     
     // Encoder declarations
     private final Encoder rotatorEncoder;
     private final Encoder flywheelEncoder;
 
     // Piston declarations
-    private final Solenoid indexerPiston;
+    // private final Solenoid indexerPiston;
 
     // Limit switches
     private final DigitalInput rotatorAtZeroSwitch;
 
 
     // Class initializer
-    public BallChucker9000(int flywheelMotorPort, int rotatorMotorPort, int indexerMotorPort, int rotatorEncoderChannelA, 
-                            int rotatorEncoderChannelB, int flywheelEncoderChannelA, int flywheelEncoderChannelB, 
-                            int indexerPistonPort, int rotatorAtZeroSwitchPort) {
-    
+    public BallChucker9000(SpeedController rotatorController, SpeedController leftFlywheelController, SpeedController rightFlywheelController, 
+                            SpeedController indexerController, int rotatorEncoderChannelA, int rotatorEncoderChannelB, int flywheelEncoderChannelA,
+                            int flywheelEncoderChannelB, int rotatorAtZeroSwitchPort) {
+
+        // PID
+        rotatorPID = new PIDController(0.5, 0, 0.5);
+        flywheelPID = new PIDController(0.5, 0, 0.5);
+
         // ESCs
-        flywheelMotor = new VictorSP(flywheelMotorPort);
-        rotatorMotor = new VictorSP(rotatorMotorPort);
-        indexerMotor = new VictorSP(indexerMotorPort);
+        leftFlywheelMotor = leftFlywheelController;
```

Comment:
The naming convention elsewhere in the code has been "...Motor" for both the field and the variable local to the constructor.